src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
```

Comment:
Consider using `@BeforeEach` or `@BeforeAll` to initialize your `RegistrationServiceImpl` instance. `@BeforeEach` runs before each test method, whereas `@BeforeAll` runs once before all test methods. Choose based on whether the state of `registrationService` should be reset for each test.